src/main/java/com/limechain/rpc/methods/system/SystemRPCImpl.java

```diff
@@ -105,9 +110,17 @@ public String[] systemLocalListenAddress() {
     /**
      * Returns currently connected peers.
      */
-    // TODO: Implement in M2.
-    public String[] systemSystemPeers() {
-        return new String[0];
+    public List<Map<String, String>> systemSystemPeers() {
+        return connectionManager
+                .getPeerIds()
+                .stream()
+                .map(connectionManager::getPeerInfo)
+                .map(peerInfo -> Map.ofEntries(
+                        entry("peerId", peerInfo.getPeerId().toString()),
+                        entry("roles", peerInfo.getNodeRoleName()),
+                        entry("bestHash", peerInfo.getBestBlockHash().toString()),
+                        entry("bestNumber", peerInfo.getBestBlock().toString())
```

Comment:
`bestNumber` should be returned as a number. From [PSP-6](https://github.com/w3f/PSPs/blob/master/PSPs/drafts/psp-6.md#15103-example):
```
{
  "jsonrpc": "2.0",
  "result": [
    {
      "bestHash": "0x603b65f208656860f7d31f494896ac2ddcff37674442a84dfbcc1de0eacd83e2",
      "bestNumber": 1193270,
      "peerId": "QmTjJKhuLXKg9CfKkgGgJrzZ7LVfSmSUkQqFfZk1prF7LE",
      "protocolVersion": 6,
      "roles": "AUTHORITY"
    },
    {
      "bestHash": "0x603b65f208656860f7d31f494896ac2ddcff37674442a84dfbcc1de0eacd83e2",
      "bestNumber": 1193270,
      "peerId": "Qme89h5f5MkdN37R173z5GSJVBVSGeUriSrp4u3Y2ZRmUv",
      "protocolVersion": 6,
      "roles": "AUTHORITY"
    }
  ],
  "id": 1
}
```

Reply:
Our response:
```
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": [
    {
      "peerId": "12D3KooWSz8r2WyCdsfWHgPyvD8GKQdJ1UAiRmrcrs8sQB3fe2KU",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    },
    {
      "peerId": "12D3KooWRjHFApinuqSBjoaDjQHvxwubQSpEVy5hrgC9Smvh92WF",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    },
    {
      "peerId": "12D3KooWK4E16jKk9nRhvC4RfrDVgcZzExg8Q3Q2G7ABUUitks1w",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    },
    {
      "peerId": "12D3KooWAsuCEVCzUVUrtib8W82Yne3jgVGhQZN3hizko5FTnDg3",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    },
    {
      "peerId": "12D3KooWGJQysxrQcSvUWWNw88RkqYvJhH3ZcDpWJ8zrXKhLP5Vr",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    },
    {
      "peerId": "12D3KooWQz2q2UWVCiy9cFX1hHYEmhSKQB2hjEZCccScHLGUPjcc",
      "bestHash": "0xcd05ffc0299876340989a32f78b08b6032e2fec2b4f4c5ef79ff3a63f2b9f8a5",
      "bestNumber": "17140817",
      "roles": "FULL"
    }
  ]
}
```